frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "dev": "npm run check-server && vite",
     "dev:msw": "cross-env API_MODE=mock vite",
     "build": "vite build --config vite-prod.config.ts",
-    "build:prod": "cross-env SHOW_DEV_PAGE=false vite build --config vite-prod.config.ts",
+    "build:prod": "vite build --config vite-prod.config.ts",
```

Comment:
We don't need this command if we're not setting the `SHOW_DEV_PAGE=false` environment variable

Reply:
I will remove it.